app/controllers/locations/map_controller.rb

```diff
@@ -0,0 +1,27 @@
+require_dependency "locations/application_controller"
+
+module Locations
+  class MapController < ApplicationController
+
+    def index
+        @map = Map.new(id: map_params[:id],loc: map_params[:loc])
+
+        if !@map.url.nil?
+          if @map.url.include? "http"
+            redirect_to @map.url
+          else
+            render plain: "#{@map.url}", status: 200
```

Comment:
@tampakis @axamei what do you think about adding our branding to marc liberation? It is a quasi-public service so I think it is worth doing. @sdellis Do all the branches have reserve landing pages like that? If yes, I'd the redirection sounds like a good way to go.